packages/simm-validation/src/is-credit-card.ts

```diff
@@ -0,0 +1,36 @@
+import { isLuhnNumber } from "./is-luhn-number";
+import { regexValidate } from "./shared/constants/regex";
+
+type CreditCardType = keyof typeof regexValidate.CREDIT_CARD;
+
+const isCreditCard = (input: any, provider?: CreditCardType) => {
+  // Returns false input is not a string
+  if (typeof input !== "string") return false;
+
+  // Remove all dashes and spaces
+  input = input.replace(/[- ]+/g, "");
```

Comment:
```suggestion
   input = input.replace(/[ -]+/g, "")
```
Ri cho không bị báo lint e